src/main/java/com/webonise/controllers/LatLngController.java

```diff
@@ -0,0 +1,48 @@
+package com.webonise.controllers;
+
+import com.webonise.models.LatLng;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LatLngController {
+
+    private ObservableList<LatLng> latLngList;
+
+    public LatLngController() {
+        latLngList = FXCollections.observableArrayList();
+    }
+
+    public ObservableList<LatLng> getLatLngList() {
+        return latLngList;
```

Comment:
Return unmodifiable list here. Use Collections.unmodifiableList().
Also you may need to provide add/remove methods.